test.js

```diff
@@ -18,3 +20,34 @@ assert(!ip.range('1.0.1.*').contains('1.0.0.10'));
 assert(ip.range('1.2.3.4-10').contains('1.2.3.4', '1.2.3.10'));
 assert(ip.range('1.2.3.5', '1.2.3.4').contains('1.2.3.4', '1.2.3.5'));
 
+// Handy tool for generating test data: http://www.ipaddressguide.com/cidr#range
+// If the range finishes with wildcards, we should have
+// one cidr notation range with a netmask.
+assert(ip.range('1.0.1.*').inetAddresses.length === 1);
+assert(ip.range('1.0.1.*').inetAddresses[0] === '1.0.1.0/24');
+
+assert(ip.range('1.0.*.*').inetAddresses.length === 1);
+assert(ip.range('1.0.*.*').inetAddresses[0] === '1.0.0.0/16');
+
+// Ranges that don't align on powers of 2 boundaries will require
+// multiple cidr notation ranges to specify.
+
+// Our package leans upon the ip-subnet-calculator module
+// to do the heavy lifting of calculating cidr notation ranges.
+// The current version over-specifies some ranges, but not so
+// much as to cause us preformance problems.  We can leave
+// eliminating redundant ranges as a possible opportunity for
+// improvement.
+assert(ip.range('1.2.3.4-10').inetAddresses.length === 3);
+assert(ip.range('1.2.3.4-10').inetAddresses.indexOf('1.2.3.4/30') !== -1);
+assert(ip.range('1.2.3.4-10').inetAddresses.indexOf('1.2.3.8/31') !== -1);
+
+// Non-contiguous ranges need multiple cidr notation ranges
+// to specify as well
+assert(ip.range('1.2.0-5.0').inetAddresses.length === 6);
+assert(ip.range('1.2.0-5.0').inetAddresses.indexOf('1.2.0.0/32') !== -1);
```

Comment:
insteado of repeating these calls, we could do with a simple `assert.equal( ip.range('1.2.0-5.0').inetAddresses, ['1.2.0.0/32', '1.2.1.0/32', ....]` here eh?